string class_name
float64 id
string[] properties
string[] values
---
bool result
string instance